### PR TITLE
Jonmv/suspend mojo

### PR DESCRIPTION
**Pull request by jonmv**
@mpolden please review.

### CODE REVIEW

File: orchestrator/src/test/java/com/yahoo/vespa/orchestrator/resources/ApplicationSuspensionResourceTest.java [hunk 1]
```diff
@@ -114,6 +115,7 @@ public void delete_works_on_suspended_and_not_suspended_applications() {
     @Test
     public void list_applications_returns_the_correct_list_of_suspended_applications() {
         // Test that initially we have the empty set
+        var v = webTarget.request(MediaType.APPLICATION_JSON);
```
Comment (mpolden) [inline orchestrator/src/test/java/com/yahoo/vespa/orchestrator/resources/ApplicationSuspensionResourceTest.java:118]: What's the purpose of this? Same for the above `v`.

Comment (jonmv) [reply orchestrator/src/test/java/com/yahoo/vespa/orchestrator/resources/ApplicationSuspensionResourceTest.java:118]: Sorry, debugging hopeless Jersey stuff. Forgot to remove.

Comment (jonmv) [reply orchestrator/src/test/java/com/yahoo/vespa/orchestrator/resources/ApplicationSuspensionResourceTest.java:118]:
```suggestion
```

File: orchestrator/src/test/java/com/yahoo/vespa/orchestrator/resources/ApplicationSuspensionResourceTest.java [hunk 2]
```diff
@@ -91,6 +91,7 @@ public void get_application_status_returns_404_for_not_suspended_and_204_for_sus
     @Test
     public void delete_works_on_suspended_and_not_suspended_applications() {
         // Delete an application that is not suspended
+        var v = webTarget.path(RESOURCE_1).request();
```
Comment (jonmv) [inline orchestrator/src/test/java/com/yahoo/vespa/orchestrator/resources/ApplicationSuspensionResourceTest.java:94]:
```suggestion
```

**Review by mpolden — APPROVED** [review]
💯

**Pull request merged**